pkg/transparentproxy/iptables/builder/builder_table_nat_test.go

```diff
@@ -16,29 +17,32 @@ var _ = Describe("Builder nat", func() {
 		func(vnet []string, verbose bool, ipv6 bool, expect ...string) {
 			// given
 			nat := tables.Nat()
-			cfg := tproxy_test.InitializeConfig(config.Config{
+			cfg := tproxy_test.InitializeConfigIPvX(config.Config{
 				Redirect: config.Redirect{
+					NamePrefix: consts.IptablesChainsPrefix,
 					Inbound: config.TrafficFlow{
-						Enabled: true,
-						Port:    1234,
-						Chain:   config.Chain{Name: "MESH_INBOUND"},
+						Enabled:           true,
+						Port:              1234,
+						ChainName:         "INBOUND",
```

Comment:
Should we add a note in `UPGRADE.md` that the chain's name has changed? If someone has automation/scripts that inject rules in the chain/after it might not work.

Reply:
Names didn't change, the logic related to adding prefix changes